Algorithms/Easy/9_PalindromeNumber/Solution.py

```diff
@@ -0,0 +1,8 @@
+class Solution(object):
+    def isPalindrome(self, x): # type x : int & return type: bool
```

Comment:
Fix the line - `def isPalindrome(self, x): # type x : int & return type: bool` like it is presented in LeetCode editor.